packages/features/users/components/UserTable/BulkActions/EventList.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Table } from "@tanstack/react-table";
+import { Check } from "lucide-react";
+import type { Dispatch, SetStateAction } from "react";
+import { useState } from "react";
+
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { SchedulingType } from "@calcom/prisma/enums";
+import { trpc } from "@calcom/trpc";
+import type { RouterOutputs } from "@calcom/trpc/react";
+import {
+  Button,
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+  Popover,
+  showToast,
+  PopoverContent,
+  PopoverTrigger,
+} from "@calcom/ui";
+
+import type { User } from "../UserListTable";
+
+interface Props {
+  table: Table<User>;
+  teams: RouterOutputs["viewer"]["organizations"]["getTeams"] | undefined;
+}
+
+export function EventListBulkAction({ table, teams }: Props) {
+  const { t } = useLocale();
+  const utils = trpc.useUtils();
+  const teamIds = teams?.map((team) => team.id);
+  const { data } = trpc.viewer.eventTypes.getByViewer.useQuery({
+    filters: { teamIds, schedulingType: SchedulingType.ROUND_ROBIN },
+  });
+  const mutation = trpc.viewer.organizations.bulkAddToEventType.useMutation({
+    onError: (error) => {
+      showToast(error.message, "error");
+    },
+    onSuccess: (res) => {
+      showToast(`${res.count} Users added to ${Array.from(selectedEvents).length} events`, "success");
```

Comment:
`res.count` produces incorrect result - if you add 1 user to 2 round robin events:

<img width="959" alt="Screenshot 2024-05-15 at 11 23 07" src="https://github.com/calcom/cal.com/assets/42170848/767e5ca8-a8ce-4407-bf8c-d5b5e8ddb673">

Then notification on bottom right corner says 2 users added:
<img width="1140" alt="Screenshot 2024-05-15 at 11 23 11" src="https://github.com/calcom/cal.com/assets/42170848/8243ad6e-fb49-4516-abc2-4f0223318ea9">

Reply:
Yes actually it return the number of host connections made, so ig a more correct message would be 2 Hosts added to 2 events. 

Or should we show number of users only?

Reply:
If you were a user, which one would be clearer for you? Personally, I think that `X users added to Y events` is the most straight-forward. If I added 1 user to 2 events and saw that `2 hosts added to 2 events` I would be bit confused.

Reply:
I agree hosts could be more confusing. Changed it to users. 

Although now if user is already a host for the event type the message will show as added 1 user, even if no changes were made.